app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java

```diff
@@ -140,8 +140,8 @@ public void onCreate(Bundle savedInstanceState) {
         ButterKnife.bind(this);
         ActivitySwitchHelper.setContext(this);
         parentLayout.setBackgroundColor(getBackgroundColor());
-        this.overridePendingTransition(R.anim.right_to_left,
-                               R.anim.left_to_right);parentLayout.setBackgroundColor(getBackgroundColor());
+        overridePendingTransition(R.anim.right_to_left,
+                R.anim.left_to_right);parentLayout.setBackgroundColor(getBackgroundColor());
```

Comment:
There's a `;` in between. Can you make the other line to appear on the next line?